lib/liquid/block_body.rb

```diff
@@ -34,15 +34,15 @@ def parse(tokenizer, parse_context, &block)
             # caller raise a syntax error
             return yield token, token
           end
-          tag_name = Regexp.last_match(1)
-          markup = Regexp.last_match(2)
+          tag_name    = Regexp.last_match(1)
+          markup      = Regexp.last_match(2)
           unless (tag = registered_tags[tag_name])
             # end parsing if we reach an unknown tag and let the caller decide
             # determine how to proceed
             return yield tag_name, markup
           end
-          new_tag = tag.parse(tag_name, markup, tokenizer, parse_context)
-          @blank &&= new_tag.blank?
+          new_tag     = tag.parse(tag_name, markup, tokenizer, parse_context)
```

Comment:
Does RuboCop force this realignment if you were to revert manually and then autocorrect after?
The reason I ask is the benefits of operator-alignment is lost when there is a block of code separating two realigned lines..

Reply:
I think the only thing that breaks the behaviour is an empty line from when I was playing around with this. In cases where the formatting isn't ideal, you need to space it.